docs/colang_2/language_reference/working-with-variables-and-expressions.rst

```diff
@@ -192,32 +195,81 @@ As in Python's formatted string literals we can use braces to evaluate an expres
 If you need to include a brace character in the literal text, it can be escaped by doubling: ``{{`` and ``}}``.
 
 ----------------------------------------
-Built-in Flow Variables
+Flow Member Variables
 ----------------------------------------
 
-.. important::
-    This is work in progress and some of the built-in variables might change or be removed in the future.
+To access a flow instance's member variables you can use a reference or the reserved variable ``$self`` from within the flow itself:
+
+.. code-block:: colang
+
+    $ref.uid: str # The unique id of the flow instance
+    $ref.flow_id: str # The name of the flow
+    $ref.status.value: str # Name of the low state ("waiting", "starting", "started", "stopping", "stopped", "finished")
+    $ref.loop_id: Optional[str] # The interaction loop id of the flow
+    $ref.parent_uid: Optional[str] # The unique id of the parent flow instance
+    $ref.child_flow_uids: List[str] # All unique ids of the child flow instances
+    $ref.context: dict # The variable context that contains all user defined variables in the flow
+    $ref.priority: float # Priority of the flow
```

Comment:
Do we have somewhere what the value means? E.g. is 0.1 "more priority" than 10000.0. Is -77.7 a valid prioritiy?

Reply:
Added more information... This still need some work in terms of making it more flexible.